src/ui_txn.c

```diff
@@ -238,6 +247,111 @@ static int step_asset_freeze_flag() {
   return 1;
 }
 
+static int step_asset_config_id() {
+  if (current_txn.asset_config.id == 0) {
+    ui_text_put("Alloc");
+  } else {
+    ui_text_put(u64str(current_txn.asset_config.id));
+  }
+  return 1;
+}
+
+static int step_asset_config_total() {
+  if (current_txn.asset_config.id != 0 && current_txn.asset_config.params.total == 0) {
+    return 0;
+  }
+
+  ui_text_put(u64str(current_txn.asset_config.params.total));
+  return 1;
+}
+
+static int step_asset_config_default_frozen() {
+  if (current_txn.asset_config.id != 0 && current_txn.asset_config.params.default_frozen == 0) {
+    return 0;
+  }
+
+  if (current_txn.asset_config.params.default_frozen) {
+    ui_text_put("Frozen");
+  } else {
+    ui_text_put("Unfrozen");
+  }
+  return 1;
+}
+
+static int step_asset_config_unitname() {
+  if (current_txn.asset_config.params.unitname[0] == '\0') {
```

Comment:
`|| current_txn.asset_config.id != 0 `

Reply:
Do you mean `&& current_txn.asset_config.id != 0`?  That's what I had there originally, but it led to a pretty clunky UI flow, confirming lots of empty or default fields for a simple invocation of `goal asset create`.

Adding `|| current_txn.asset_config.id != 0` doesn't really matter because those transactions already have those fields all set to zero anyway (since those fields cannot be changed after asset creation).

Reply:
I did mean `||`, my thinking being that because those fields won't have an effect if `id != 0`, we might as well not display them... but thinking about it more it's not worth cluttering the code for that